index.js

```diff
@@ -81,15 +55,23 @@ function processArgv(argv) {
       break;
   }
 
-  // CLI handles new command without passing through to modules
-  if (passToModules) {
-    if (fs.existsSync(packageJsonPath)) {
-      const modules = getModules(require(packageJsonPath));
-      runCommand(modules, command, argv);
+  // Look globally and locally for matching glob pattern
+  const dirs = [
+    `${process.cwd()}`, // local (where they ran the command from)
+    `${__dirname}/..`   // global (where this code exists)
+  ];
+
+  getGlobs(dirs).forEach(pkg => {
+    const module = require(path.dirname(pkg));
```

Comment:
We should probably wrap the `require` in a `try/catch` in case there are invalid modules installed. To test the failure, create a folder **node_modules/@blackbaud/skyux-builder-foobar**. Add a package.json file with only the `name` set (ie., _no_ index.js file). You should see:

```
Error: Cannot find module...
```
I had to do something similar with the plugins feature: https://github.com/blackbaud/skyux-builder/blob/master/loader/sky-processor/index.js#L17

Reply:
(This failure was caught because this repo was installed locally: blackbaud/skyux-builder-config) <-- No index.js file

Reply:
Good catch.  Will adjust.